tests/OrbitStubClientTest.php

```diff
@@ -29,11 +29,25 @@ public function testGetContentWithPresentButUnusedConstructor()
         );
 
         $expectedOrbit = new Orbit(
-            '<orbit-head/>',
+            '<orbit-head><orbit-template-params data-values="[]"/></orbit-head>',
             '<orbit-bodyfirst/>',
             '<orbit-bodylast/>'
         );
 
         $this->assertEquals($expectedOrbit, $orbitClient->getContent());
     }
+
+    public function testGetContentWithTemplateParameters()
+    {
+        $orbitClient = new OrbitStubClient();
+
+        $expectedOrbit = new Orbit(
+            '<orbit-head><orbit-template-params data-values="'
+                    . htmlentities(json_encode(['testKey' => 'testValue'])) .'"/></orbit-head>',
```

Comment:
no preference about this point, I thought the first one was not easy to read so I decided to use the encoding functions. I'm going to update it with the raw value, thanks for the suggestion